> 0:15:16 /home/buildnode/jenkins/workspace/swift-PR-Linux/branch-master/swiftpm/Tests/PackageLoadingTests/PD4_2LoadingTests.swift:392: error: PackageDescription4_2LoadingTests.testCaching : XCTAssertEqual failed: ("[<AbsolutePath:"/tmp/spm-tests-testCaching.H9akuF/pkg/Package.swift">]") is not equal to ("[]") - 
> 10:15:16 /home/buildnode/jenkins/workspace/swift-PR-Linux/branch-master/swiftpm/Tests/PackageLoadingTests/PD4_2LoadingTests.swift:392: error: PackageDescription4_2LoadingTests.testCaching : XCTAssertEqual failed: ("[<AbsolutePath:"/tmp/spm-tests-testCaching.H9akuF/pkg/Package.swift">]") is not equal to ("[]") - 

Hm, there could be a correctness issue with one of the high-level operations. Weird that it only happens on Linux, though.

--- a/stdlib/public/core/HashTable.swift
+++ b/stdlib/public/core/HashTable.swift
@@ -94,7 +94,6 @@ extension _HashTable {
     for object: AnyObject,
     scale: Int8
   ) -> Int {
-#if false // FIXME: Enable per-instance seeding
     // We generate a new hash seed whenever a new hash table is allocated and
     // whenever an existing table is resized, so that we avoid certain copy
     // operations becoming quadratic.  (For background details, see
@@ -115,10 +114,6 @@ extension _HashTable {
     // guarantee that no two tables with the same seed can coexist at the same
     // time (apart from copy-on-write derivatives of the same table).
     return unsafeBitCast(object, to: Int.self)
-#else
-    // Use per-capacity seeding for now.
-    return Int(scale)
-#endif
   }
 }
 

